cmd/nodepool/aws/create.go

```diff
@@ -0,0 +1,111 @@
+package aws
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
+	"github.com/openshift/hypershift/cmd/log"
+	"github.com/openshift/hypershift/cmd/nodepool/core"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/types"
+	crclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type AWSPlatformCreateOptions struct {
+	InstanceProfile string
+	SubnetID        string
+	SecurityGroupID string
+	InstanceType    string
+	RootVolumeType  string
+	RootVolumeIOPS  int64
+	RootVolumeSize  int64
+}
+
+func NewCreateCommand(coreOpts core.CreateNodePoolOptions) *cobra.Command {
+	platformOpts := AWSPlatformCreateOptions{
+		InstanceType:   "m4.large",
```

Comment:
Let's update as in https://github.com/openshift/hypershift/pull/850

Reply:
Also https://github.com/openshift/hypershift/pull/852

Reply:
Done